src/pages/publication/PublicProfile.tsx

```diff
@@ -61,14 +61,18 @@ const PublicProfile: FC = () => {
       <StyledUserInfo>
         <Card>
           <Heading>{user.name}</Heading>
-          <LabelTextLine label={t('heading.organizations')}>
-            {isLoadingMultiple ? (
-              <CircularProgress color="inherit" size={20} />
-            ) : (
-              units?.map((unit) => <NormalText key={unit.id}>{unit.name}</NormalText>)
-            )}
-          </LabelTextLine>
-
+          {isLoadingMultiple ? (
+            <CircularProgress color="inherit" size={20} />
```

Comment:
hva trenger vi egentlig vår `Progress` til når vi kan gjøre det så enkelt?

Reply:
hahaha, la ikke merke til at jeg ikke brukte vår en gang 😆

Reply:
kanskje vi bare skal fjerne `Progress` ?

Reply:
ser ut som vi kan gjøre det iaff... men lag egen sak for det i så fall da